pkgs/unified_analytics/lib/src/enums.dart

```diff
@@ -70,21 +63,34 @@ enum DashEvent {
   });
 }
 
-/// Officially-supported clients of this package.
+/// Officially-supported clients of this package as logical
+/// tools, grouped by user point of view.
 ///
 /// All [label] values should use an underscore as a delimiter.
 enum DashTool {
-  dartTools(
-    label: 'dart_tools',
-    description: 'A CLI for Dart development',
+  dartTool(
+    label: 'dart-tool',
+    description: 'Dart CLI developer tool',
   ),
-  flutterTools(
-    label: 'flutter_tools',
+  flutterTool(
+    label: 'flutter-tool',
     description: 'Runs flutter applications from CLI',
   ),
-  languageServer(
-    label: 'language_server',
-    description: 'The Dart language server for IDE and CLI support.',
+  vscodePlugins(
+    label: 'vscode-plugins',
+    description: 'VS Code IDE plugins for Dart and Flutter',
+  ),
+  intellijPlugins(
+    label: 'intellij-plugins',
+    description: 'IntelliJ IDE plugins for Dart and Flutter',
+  ),
+  androidStudioPlugins(
+    label: 'android-studio-plugins',
+    description: 'Android Studio IDE plugins for Dart and Flutter',
+  ),
+  devtools(
+    label: 'devtools',
+    description: 'DevTools debugging and performance tools',
```

Comment:
Small nit but can we add these in alphabetical order?

Reply:
I can.  I ordered them the order they are in in the PDD for my peace of mind, but fortunately that isn't a requirement :-D